recipes/pulsar_phase/pulsar_phase_computation.ipynb

```diff
@@ -0,0 +1,1895 @@
+{
```

Comment:
<p>It would be nice to add comments on the lines to explain what it means when it is not explicit (e.g. TZRSITE is not clear to me...)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy-recipes/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Reply:
I mean this is the format of ephemeris file, so is it really our job to explain that ?

We could put a link to the documentation pdf of Tempo2, which kind of explain all this. But isn't it going to be confusing to refer to Tempo2 while using PINT ...

Reply:
<p>In my opinion this is something that should be described in Tempo2/PINT, although I agree their documentation is not very clear sometimes. The documentation that @MRegeard is talking about explains everything and could be good option to mention although we do not use tempo2.</p>